deploy/docker/canary/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:latest
+MAINTAINER vishnuk@google.com
+
+
+RUN apt-get install -y git
+RUN git clone https://github.com/kubernetes/heapster.git /go/src/k8s.io/heapster
+RUN cd /go/src/k8s.io/heapster && make
+
+ENTRYPOINT ["/go/src/k8s.io/heapster/heapster"]
```

Comment:
You're right. In that case, instead of git clone you can do `go get -u -d k8s.io/heapster` instead of the git clone, but essentially it's the same.